app/src/main/java/org/notests/rxfeedbackexample/github/paginated/search/GithubPaginatedSearch.kt

```diff
@@ -328,16 +327,15 @@ fun parseLinks(links: String): Map<String, String> {
     return result
 }
 
-private fun GithubPaginatedSearchActivity.showOrHideError(error: Optional<GitHubServiceError>) {
-    if (error is Optional.Some) {
-        Toast.makeText(this, error.data.displayMessage, Toast.LENGTH_SHORT).show()
+private fun GithubPaginatedSearchActivity.showOrHideError(error: GitHubServiceError?) {
+    error?.let {
```

Comment:
As I said in comment before - when we reach this place error would never be null because chain will throw NPE before (it cannot pass null) and Driver will swallow it (it looks like it works but it only works because of Driver)